salsah/src/public/js/jquery.resadd.js

```diff
@@ -517,6 +517,7 @@
 												}
 											}
 											tmpele.attr('placeholder', placeholderText);
+											tmpele.attr('autocomplete', 'off');
```

Comment:
I did add it.  

As you know it, autocomplete is a feature of the browser, it does the same as the searchbox does: suggesting results as the user is typing.  

And therefore, they are concurrent by behaving the same way.  

Disabling `autocomplete` is a directive to the browser not to store and propose previous choices.

With `autocomplete` enabled, the browser keeps a list of previous values and suggests the matching ones. So if I searched for "thi" before, it will pop-down a list with "thi", hiding the results of the `searchbox`.

I added it, I don't know how, but salsah v1.0 didn't try to autocomplete.